tools/wpt/testfiles.py

```diff
@@ -396,7 +397,8 @@ def run_changed_files(**kwargs):
     separator = "\0" if kwargs["null"] else "\n"
 
     for item in sorted(changed):
-        sys.stdout.write(os.path.relpath(item.encode("utf8"), wpt_root) + separator)
+        assert isinstance(item, six.text_type)
+        sys.stdout.write(os.path.relpath(item, wpt_root) + separator)
```

Comment:
This seems like it regresses Python 2 by missing an encoding step there.

Reply:
Well the `encode()` was added in https://github.com/web-platform-tests/wpt/commit/e98eee5aec6ba154f2f086e9d2f0a287617e8519 and I still don't see the rationale of the change. @gsnedders maybe?

Reply:
Yeah, this results in `tools/wpt/testfiles.py:401: error: Argument 1 to "write" of "IO" has incompatible type "unicode"; expected "str"` from MyPy in Py2 mode.

Reply:
So it seems like here is where we need to `ensure_str` since this is the edge of the system where it's (relatively) OK to bifurcate the types to appease the stdlib requirements.